### PR TITLE
Resume on GSOC19

### PR DESCRIPTION
**Pull request by jorgepiloto**
This will add the corresponding resume on my GSOC19 at `poliastro`. It includes:

* Achieved implementations.
* `#TODO` implementations.
* Personal experience at GSOC.

### CODE REVIEW

File: content/articles/2019-08-20-new-features-after-gsoc19.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+### Izzo Algorithm minor errors
+
+Some errors appeared in the minimum transfer time for the Izzo algorithm. After
+a little bit of debugging, Juanlu found that `time-of-flight equation that does
+not need external computation of one of the terms`. I was asked to write some
+unit tests.
+
+<center>
+**[GET THE CODE HERE](https://github.com/poliastro/poliastro/pull/709)**
```
Comment (astrojuanlu) [inline content/articles/2019-08-20-new-features-after-gsoc19.md]: The only thing that I dislike is the `GET THE CODE HERE` link in all caps :) Can we have something like `link to the corresponding PRs` ?

**Review by astrojuanlu — COMMENTED** [review]
The summary is superb and I just left one style comment. With that we can merge :)

File: content/articles/2019-08-20-new-features-after-gsoc19.md (hunk 1 above)
Comment (jorgepiloto) [reply content/articles/2019-08-20-new-features-after-gsoc19.md]: Sure! :tada:

**Review by astrojuanlu — APPROVED** [review]

**Comment by astrojuanlu** [comment]
Merged!

**Pull request merged**